mw-webapp/src/component/header/HeaderContext.ts

```diff
@@ -0,0 +1,20 @@
+import {createContext, useContext} from "react";
+import {User} from "firebase/auth";
+
+const STUB_USER = {} as User;
```

Comment:
hey!
no as operators

Reply:
So what can I use if as and null is not allowed?